server.js

```diff
@@ -79,9 +79,7 @@ app.delete('/api/v1/teams/:id', (request, response) => {
   const teamID = request.params.id;
 
   database('teams').where('id', teamID).del()
-    .then(() => {
```

Comment:
ah, i should have changed this back when you mentioned it! i totally forgot. thank you!